week1s/task1/style.css

```
@@ -0,0 +1,45 @@
+body, ul {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: palegreen;
+    background-image: url('./images/dog-bbg.jpg');
+}
+
+div {
+    width: 600px;
+    height: 500px;
```

너비와 높이가 같아야 border-radius: 50%를 했을 때 동그란 원 모양이 됩니다!
```suggestion
    width: 600px;
    height: 600px;
```